paperqa/clients/journal_quality.py

```diff
@@ -44,7 +44,12 @@ async def _process(
         # remember, if both have docnames (i.e. key) they are
         # wiped and re-generated with resultant data
         return doc_details + DocDetails(  # type: ignore[call-arg]
-            source_quality=self.data.get(query.journal.casefold(), -1)  # type: ignore[union-attr]
```

Comment:
Should we be using `casefold`?

Reply:
I think so? just to be more robust with journal names. @mskarlin  thoughts?

Reply:
yup that's what we've been using on title comparisons!